sdk/eventhub/event-processor-host/src/hostContext.ts

```diff
@@ -13,7 +13,7 @@ import {
   EventHubPartitionRuntimeInformation,
   EventHubConnectionConfig
 } from "@azure/event-hubs";
-import AsyncLock from "async-lock";
+import AsyncLock from 'async-lock';
```

Comment:
How is this change related to any of the warnings?

Reply:
It was, though I can't remember what tool warned about it now that I think about it.